pkg/scaffold/resource/crd_sample.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"strings"
+
+	"sigs.k8s.io/controller-tools/pkg/scaffold/input"
+)
+
+var _ input.File = &CRD{}
+
+// CRDSample scaffolds a manifest for CRD sample.
+type CRDSample struct {
+	input.Input
+
+	// Scope is Namespaced or Cluster
+	Scope string
```

Comment:
Creating a type for Scope instead of plain string?

Reply:
removed scope because we aren't using it in this case.